assets/src/edit-story/app/story/effects/useHistoryEntry.js

```diff
@@ -22,8 +23,9 @@ function useHistoryEntry( {
 			current,
 			pages,
 			selection,
+			capabilities,
 		} );
-	}, [ appendToHistory, story, current, pages, selection ] );
+	}, [ appendToHistory, story, current, pages, selection, capabilities ] );
```

Comment:
Actually, (I might be missing something but) why are capabilities relevant to history entry? Aren't capabilities static and coming from DB, and "Undo" and "Redo" shouldn't influence the capabilities? Thoughts?

Reply:
ATM capabilities are stored in the story state. If capabilities are not copied when state is changed, then the capabilities lost and the error you saw happens. 

That is an argument to be said that they shouldn't be stored there in the first place. That is another discussion.

Reply:
Editor-wide state was a topic the other day already. Thoughts, @barklund?

Reply:
I did look at this one, and didn't really know how to feel about it. On one hand, it doesn't feel like something that belongs inside the story reducer, as it's not in any way part of the story nor is it anything you can edit. But one the other hand, where else would it go?

I would probably prefer this being taken back out if the story reducer, but not completely sure about where to put it instead. Maybe it's an API level thing and belongs in the API state? It's a tricky one.

Reply:
Sounds good to me to merge as-is for the purposes of this issue, and open a separate ticket for discussing how to store it "properly".

Reply:
sgtm.